Update TABLE.md

pabloalmunia opened the pull request:
- fix minor error: "method name" is changed with "property name"
- add `Initializers` descriptor.

pabloalmunia committed:
Update TABLE.md

- fix minor error: "method name" is changed with "property name"
- add `Initializers` descriptor.

littledan commented on TABLE.md:
```diff
@@ -62,4 +62,6 @@
 
 <sup>11</sup> `finisher` function is a callback that is called at the end of class creation. It's optional.
 
+<sup>12</sup> `kind: "initializer"` permits to define a `finisher` callback to be used purely for perform a side effect when instantiating a class. This descriptor don't create new members. Do not confuse with the `initializer` property used to set the initial value to a field.
```
I don't think we should describe initializers as finishers, as the timing is different. Instead, let's describe them as field-like things that don't define anything.

pabloalmunia committed:
Update TABLE.md

pabloalmunia committed:
Update TABLE.md

pabloalmunia commented:
It's true, perhaps it's bit confused, but the field with the callback for the side effect is included into `finisher` field of this return descriptor.

I change the text for a clearest description.

pabloalmunia committed:
Update TABLE.md

Fix a very important mistake. The `kind: "initializer"` use the field `initializer` for the callback and I wrote the field for this callback was `finisher`. Apologize for my confusion.

pabloalmunia commented:
Fixed a very important mistake. The `kind: "initializer"` use the field `initializer` for the callback and I wrote the field for this callback was `finisher`. Apologize for my confusion.

littledan commented:
Thanks for this quick documentation!
